addon/services/flash-messages.js

```diff
@@ -68,6 +68,13 @@ export default Service.extend({
     return this;
   },
 
+  getFlashObject() {
```

Comment:
Awesome!

I'm thinking of maybe having all 3 methods.  I like the readability of: 
`flash.add({ message: 'foo' }).getFlashObject();` as it returns the object we just added. `peekLast` doesn't make sense to chain.

Then we can still have `peekLast` and `peekFirst` but encourage that they aren't chainable but are public calls to our service.

Thoughts?